scipy/stats/tests/test_discrete_distns.py

```diff
@@ -29,6 +29,15 @@ def test_hypergeom_logpmf():
     assert_almost_equal(hypergeom_logpmf, bernoulli_logpmf, decimal=12)
 
 
+def test_nhypergeom_logpmf():
+    # test with hypergeom
+    N, K, r = 45, 13, 8
+    k = 6
+    NHG = nhypergeom._pmf(k, N, K, r)
+    HG = hypergeom._pmf(k, N, K, k+r-1) * (N - K - (r-1)) / (N - (k+r-1))
```

Comment:
Use the standard, public `pmf` method.
```suggestion
    NHG = nhypergeom.pmf(k, N, K, r)
    HG = hypergeom.pmf(k, N, K, k+r-1) * (N - K - (r-1)) / (N - (k+r-1))
```

Reply:
Sorry, forgot to change that. Done that now!